src/main/resources/hudson/tasks/test/TestResultProjectAction/jobMain.jelly

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:t="/lib/hudson" xmlns:test="/lib/hudson/test">
     <table style="margin-top: 1em; margin-left: 1em;">
-        <j:set var="tr" value="${it.job.lastCompletedBuild.getAction(it.class.classLoader.loadClass('hudson.tasks.test.AbstractTestResultAction'))}"/>
+        <j:set var="tr" value="${it.job.lastCompletedBuild.getAction(it.testResultActionClass)}"/>
```

Comment:
@oleg-nenashev i still think it's fine, as we're looking not for `it` instances (which is test result *project* action) but for linked test result actions.